### ISSUE TITLE
Icon request: timer

### ISSUE THREAD

**Issue by danieldaeschle** [feathericons/feather#735]
A timer icon (or stopwatch) which is like clock but with buttons on the upside. Timer icon also exists on material design icons.

**Comment by locness3** [comment]
Please make a new issue and use the icon request template. Thanks :)

**Comment by ahtohbi4** [comment]
Duplicate of #753

**Comment by locness3** [comment]
#753 was created after this one.

**Comment by ahtohbi4** [comment]
Yeah, but #753 is matching to a template of icon request issue, unlike this one. 🙂

**Comment by locness3** [comment]
Yeah, but that doesn't make it the first issue requesting the icon.

**Comment by ahtohbi4** [comment]
I believe that the date of creation of the issue is a less important thing than the completeness of the technical specification.
